src/test/incompatiblePropertiesTest.ts

```diff
@@ -0,0 +1,24 @@
+import * as assert from "assert"
+import * as path from "path"
+import * as index from "../index"
```

Comment:
I think its not a great pattern to import index with `*`, in general its better to import only what you need but in the index case in particular

Reply:
Currently done this way in every test.  I can change all tests in a separate PR.  Do you recommend this instead?

```
import { OpenApiDiff } from "../index"

const diff = new OpenApiDiff({})
```

Reply:
Or I can remove all `import *` if you think it's best.

Reply:
I think yeah it’s best practice to not import *, it doesn’t matter as much in test but in the code it allows bundled to only pick what they need and produce minimal packages. However you prefer.
If you also don’t need the .js extension to import you can probably omit index and just do import … from “..”

Reply:
https://github.com/Azure/openapi-diff/issues/319